command/v6/delete_orphaned_routes_command_test.go

```diff
@@ -149,136 +147,45 @@ var _ = Describe("deleted-orphaned-routes Command", func() {
 
 						It("returns an error", func() {
 							Expect(executeErr).To(HaveOccurred())
-
-							Expect(fakeActor.GetOrphanedRoutesBySpaceCallCount()).To(Equal(0))
-							Expect(fakeActor.DeleteRouteCallCount()).To(Equal(0))
+							Expect(fakeActor.DeleteUnmappedRoutesCallCount()).To(Equal(0))
 						})
 					})
 
 					When("the user inputs yes", func() {
-						var routes []v2action.Route
-
 						BeforeEach(func() {
 							_, err := input.Write([]byte("y\n"))
 							Expect(err).NotTo(HaveOccurred())
-
-							routes = []v2action.Route{
-								{
-									GUID: "route-1-guid",
-									Host: "route-1",
-									Domain: v2action.Domain{
-										Name: "bosh-lite.com",
-									},
-									Path: "/path",
-								},
-								{
-									GUID: "route-2-guid",
-									Host: "route-2",
-									Domain: v2action.Domain{
-										Name: "bosh-lite.com",
-									},
-								},
-							}
-
-							fakeActor.GetOrphanedRoutesBySpaceReturns(routes, nil, nil)
 						})
 
 						It("displays getting routes message", func() {
```

Comment:
Please update this test description to reflect changes to the output (it's no longer printing a message about getting routes)